src/schemapack/_internals/exceptions.py

```diff
@@ -204,3 +204,44 @@ class CircularRelationError(BaseError, ValueError):
     """Raised when a circular relation between resources is detected, but the requested
     operation cannot be performed on datapacks with circular relations.
     """
+
+
+class ClassNotFoundError(BaseError, KeyError):
+    """Raised when a class was not found in a schemapack or datapack."""
+
+    def __init__(self, *, class_name: str, in_schemapack: bool):
```

Comment:
Maybe a `ModelType` enum would be a bit more self-explanatory than the `in_schemapack` bool.

Reply:
I agree, thanks